esmvaltool/cmor/table.py

```diff
@@ -141,6 +145,23 @@ def _load_coordinates(self):
                     coord.read_json(table_data['axis_entry'][coord_name])
                     self.coords[coord_name] = coord
 
+    def _load_control_vocabulary(self):
```

Comment:
I think this should be called `_load_controlled_vocabulary` in accordance with https://github.com/WCRP-CMIP/CMIP6_CVs